itl/plugins-contrib.d/web.conf

```diff
@@ -561,3 +561,28 @@ object CheckCommand "ssl_cert" {
 	vars.ssl_cert_address = "$check_address$"
 	vars.ssl_cert_port = 443
 }
+
+object CheckCommand "varnish" {
+    import "plugin-check-command"
+    command = [ PluginDir + "/check_varnish" ]
+
+    arguments = {
+        "-n" = {
+            value = "$varnish_name$"
+            description = "Specify the Varnish instance name"
+        }
+        "-p" = {
+            value = "$varnish_param$"
+            description = "Specify the parameter to check (see below). The default is 'ratio'."
+        }
+        "-c" = {
+            value = "$varnish_critical$"
+            description = "Set critical threshold: [@][lo:]hi"
+        }
+        "-w" = {
+            value = "$varnish_warning$"
+            description = "Set warning threshold: [@][lo:]hi"
+        }
+    }
+}
```

Comment:
Please correct the indent to use tabs, not spaces.

Reply:
done